src/commands/flinkStatements.ts

```diff
@@ -195,7 +201,7 @@ export async function submitFlinkStatementCommand(
     computePool,
     properties: new FlinkSpecProperties({
       currentDatabase,
-      currentCatalog: computePool.environmentId,
+      currentCatalog: currentDatabaseKafkaCluster.environmentId,
```

Comment:
Unrelated to this PR, but seemed like it needed to be handled before something slipped through

Reply:
They ought to be the same env id, AFAICS, but in case they aren't, erring on side of kafka cluster's is more appropriate, thanks.